docs/getting-started/fundamentals/javascript-in-the-block-editor.md

```diff
@@ -75,7 +75,7 @@ wp.blocks.registerBlockVariation(
 
 For scripts that need to run in the Block Editor, make sure you use the [`enqueue_block_editor_assets`](https://developer.wordpress.org/reference/hooks/enqueue_block_editor_assets/) hook coupled with the standard [`wp_enqueue_script`](https://developer.wordpress.org/reference/functions/wp_enqueue_script/) function.
 
-Refer to [Enqueueing assets in the Editor](/docs/how-to-guides/enqueueing-assets-in-the-editor.md) for more information. You can also visit the [block-development-example](https://github.com/wptrainingteam/block-theme-examples/blob/master/example-block-variation/functions.php) GitHub repository for more practical examples.
+Refer to [Enqueueing assets in the Editor](/docs/how-to-guides/enqueueing-assets-in-the-editor.md) for more information. You can also visit the [block-development-examples](https://github.com/WordPress/block-development-examples) GitHub repository for more practical examples.
```

Comment:
```suggestion
Refer to [Enqueueing assets in the Editor](/docs/how-to-guides/enqueueing-assets-in-the-editor.md) for more information.
```

I think this sentence can be removed. This section is expected to link to specific documentation on enqueuing assets in the block editor, but [the Enqueueing assets in the Editor](https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/enqueueing-assets-in-the-editor.md) page itself is already packed with that information.